NotebookExamples/csharp/Samples/DataFrame-Getting Started.ipynb

```diff
@@ -0,0 +1,387 @@
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "// To change a value directly through df\n",
+    "df[0, 1] = 10; // 0 is the rowIndex, and 1 is the columnIndex. This sets the 0th value in the Ints columns to 10\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table><thead><th><i>index</i></th><th>DateTimes</th><th>Ints</th><th>Strings</th></thead><tbody><tr><td>0</td><td>2019-01-01 00:00:00Z</td><td>10</td><td>&lt;null&gt;</td></tr><tr><td>1</td><td>2019-01-01 00:00:00Z</td><td>100</td><td>Foo!</td></tr><tr><td>2</td><td>2019-01-01 00:00:00Z</td><td>&lt;null&gt;</td><td>&lt;null&gt;</td></tr></tbody></table>"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "// Modify ints and strings columns by indexing\n",
+    "ints[1] = 100;\n",
+    "strings[1] = \"Foo!\";\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<span><a href=\"https://docs.microsoft.com/dotnet/api/System.Int32?view=netcore-3.0\">System.Int32</a></span>"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "// Indexing can throw when types don't match.\n",
+    "// ints[1] = \"this will throw because I am a string\";  \n",
+    "// DataType can be used to figure out the type of data in a column. \n",
+    "ints.DataType"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<table><thead><th><i>index</i></th><th>DateTimes</th><th>Ints</th><th>Strings</th></thead><tbody><tr><td>0</td><td>2019-01-01 00:00:00Z</td><td>15</td><td>&lt;null&gt;</td></tr><tr><td>1</td><td>2019-01-01 00:00:00Z</td><td>105</td><td>Foo!</td></tr><tr><td>2</td><td>2019-01-01 00:00:00Z</td><td>&lt;null&gt;</td><td>&lt;null&gt;</td></tr></tbody></table>"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "// Add 5 to ints through the DataFrame\n",
+    "df[\"Ints\"].Add(5, inPlace: true);\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "Unhandled exception",
+     "evalue": "(4,3): error CS0650: Bad array declarator: To declare a managed array the rank specifier precedes the variable's identifier. To declare a fixed size buffer field, use the fixed keyword before the field type.\r\n(4,4): error CS0270: Array size cannot be specified in a variable declaration (try initializing with a 'new' expression)",
```

Comment:
what's the deal here?

![image](https://user-images.githubusercontent.com/8291187/69350381-126fa100-0c3f-11ea-8d1a-a12e21a2036c.png)

Reply:
Unrelated error message to the cell. Cleared now.